src/modules/apps/e2e/report/appId.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { Box, Color } from 'ink'
+import { parseAppId } from '@vtex/api'
+
+
+interface AppIdProps {
+    appId: string
+  }
+  
+  export const AppId: React.FunctionComponent<AppIdProps> = ({ appId }) => {
+    const { name, version, build } = parseAppId(appId)
+    const [vendor, app] = name.split('.')
+    return (
+      <Box>
+        <Color blue>{vendor}</Color>
+        <Color dim>.</Color>
+        {`${app}@${version}`}
+        {build && <Color dim>{`+${build}`}</Color>}
+      </Box>
+    )
+  }
```

Comment:
Missing formatting here, I think you'll have to add .tsx files to:
https://github.com/vtex/toolbelt/blob/fd67cc862b3c69ef5e0fe8792fc38c6ed37cd520/package.json#L10-L11
And
https://github.com/vtex/toolbelt/blob/fd67cc862b3c69ef5e0fe8792fc38c6ed37cd520/package.json#L19

As an improvement to the prettier one they could be something like:
``` 
lint: 'eslint ./src --cache --ext js,jsx,ts,tsx --config .eslintrc',
format: 'prettier --config ./.prettierrc --write "./src/**/*.{ts,tsx,js,jsx,json}"',
```